Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'bcrypt', '~> 3.1.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+# Features an easy-to-use API
+gem 'rexml', '~> 3.2.4'
```

Comment:
Because i am getting this error when the tests are loaded:
```
Failure/Error: require File.expand_path('../config/environment', __dir__)

LoadError:
  cannot load such file -- rexml/document

```
But now I moved to test group